spacepy/coordinates.py

```diff
@@ -728,23 +728,13 @@ def quaternionFromMatrix(matrix, scalarPos='last'):
                            np.identity(3), atol=0.2):
             raise ValueError('Input rotation matrix{} not orthogonal.'.format(
                 '' if len(matrix.shape) == 2 else ' at ' + str(i)))
-        try:
-            det = np.linalg.det(m)
-        except AttributeError: # det new in numpy 1.8.0
-            det = m[0, 0] * m[1, 1] * m[2, 2] \
-                  + m[0, 1] * m[1, 2] * m[2, 0] \
-                  + m[0, 2] * m[1, 0] * m[2, 1] \
-                  - m[0, 2] * m[1, 1] * m[2, 0] \
-                  - m[0, 1] * m[1, 0] * m[2, 2] \
-                  - m[0, 0] * m[1, 2] * m[2, 1]
+        det = np.linalg.det(m)
```

Comment:
Excellent catch here!

Reply:
That was an easy one since I remember writing it pretty recently :)